_pages/dev/ssr/server-side-rendering-in-spartacus.md

```diff
@@ -21,26 +21,28 @@ The following steps describe how to manually add SSR support so that your Sparta
 1. Add the following dependencies to `package.json`:
 
     ```json
-    "@angular/platform-server": "~8.2.14",
-    "@nguniversal/express-engine": "^8.1.1",
-    "@nguniversal/module-map-ngfactory-loader": "^8.2.6",
+    "@angular/platform-server": "~10.1.0",
+    "@nguniversal/express-engine": "^10.1.0",
     "express": "^4.15.2"
     ```
 
 1. Add the following developer dependencies to `package.json`:
 
     ```json
-    "ts-loader": "^5.3.2",
-    "webpack-cli": "^3.3.2"
+    "ts-loader": "^6.0.4",
+    "webpack-cli": "^3.3.12",
+    "@nguniversal/builders": "^10.1.0",
+    "@types/express": "^4.17.0",
     ```
 
 1. For convenience, add the following scripts to `package.json`:
 
     ```json
-    "compile:server": "webpack --config webpack.server.config.js --progress --colors",
-    "serve:ssr": "node dist/server",
-    "build:ssr": "npm run build:client-and-server-bundles && npm run compile:server",
-    "build:client-and-server-bundles": "ng build --prod && ng run spartacus:server"
+    "e2e": "ng e2e",
+    "dev:ssr": "ng run test:serve-ssr",
+    "serve:ssr": "node dist/test/server/main.js",
+    "build:ssr": "ng build --prod && ng run test:server:production",
+    "prerender": "ng run test:prerender"
```

Comment:
Leaving 'test' in instruction without any notification for a change might cause some misunderstanding or problems. Maybe let's keep with a pattern of `<your-project-name>` and Note about changing this part.

Reply:
yeap, not the best name for a test app :)